EU4toV2/Source/Mappers/PartyTypes/PartyType.h

```diff
@@ -30,11 +34,7 @@ class PartyType: commonItems::parser
 	date start_date;
 	date end_date;
 
-	std::string economic_policy;
-	std::string trade_policy;
-	std::string religious_policy;
-	std::string citizenship_policy;
-	std::string war_policy;
+	std::map<std::string, std::string> policies;
```

Comment:
Add comment behind:
// policyName / policy position 
So we know what's what in the map.